cpp/benchmarks/io/json/json_reader_input.cpp

```diff
@@ -123,3 +152,10 @@ NVBENCH_BENCH_TYPES(BM_json_read_io, NVBENCH_TYPE_AXES(io_list))
   .set_name("json_read_io")
   .set_type_axes_names({"io"})
   .set_min_samples(4);
+
+NVBENCH_BENCH_TYPES(BM_json_read_compressed_io,
+                    NVBENCH_TYPE_AXES(compression_list, nvbench::enum_type_list<io_type::FILEPATH>))
+  .set_name("json_read_compressed_io")
+  .set_type_axes_names({"compression_type", "io"})
+  .add_int64_power_of_two_axis("data_size", nvbench::range(20, 29, 1))
```

Comment:
Q: Did we mean to do 31 instead of 1?

```suggestion
  .add_int64_power_of_two_axis("data_size", nvbench::range(20, 29, 31))
```

Reply:
No, the [nvbench range](https://github.com/NVIDIA/nvbench/blob/main/docs/benchmarks.md#nvbenchrange) function takes stride as the third parameter.

Reply:
Ah I see, I was interpreting it as a list. Thanks for clarifying!